dnsdbq.c

```diff
@@ -714,19 +706,19 @@ help(void) {
 	printf("usage: %s [-acdfGghIjmqSsUv468] [-p dns|json|csv|minimal]\n",
 	       program_name);
 	puts("\t[-u SYSTEM] [-V VERB] [-0 FUNCTION=INPUT]\n"
-	     "\t[-k (first|last|duration|count|name|data)[,...]]\n"
+	     "\t[-k (first|last|duration|count|name|type|data)[,...]]\n"
```

Comment:
why is rrtype specified as "type"?  I thought we use "rrtype" consistently.

Reply:
i'm going to keep it as 'type' as in <name,class,type>.